packages/p2p/src/components/online-status/online-status-label.tsx

```diff
@@ -15,60 +16,68 @@ const OnlineStatusLabel = ({
     last_online_time,
     size = isMobile() ? 'xxxs' : 'xs',
 }: TOnlineStatusLabelProps) => {
+    const getDifference = (last_seen_online: number) => {
+        const start_time = epochToMoment(last_seen_online).unix();
+        const end_time = toMoment().unix();
+        return getDiffDuration(start_time, end_time);
+    };
+
+    const getStatusLabel = (diff: Duration) => {
+        if (diff.years()) return localize('Seen more than 6 months ago');
```

Comment:
@farhan-nurzi-deriv i understand that this can be moved to utils folder but since it's not reused by others, let's leave it here for now. We'll refactor the utility functions as well.